test/run/test.utils.js

```diff
@@ -16,6 +16,17 @@ var which = require("which");
 
 var prevDir, prevBinary;
 
+sandbox.configure({
+  globals: {
+    // As of Node 12.x, the process global is no longer an enumerable property of the global.
+    // https://github.com/nodejs/node/pull/26882
+    // Consequently, sandboxed-module is unable to find the global. To fix this, explicitly
+    // export the "process" global, so that modules (e.g. "which") that use "process" are
+    // able to obtain the "process" global from the sandbox.
+    process: process,
```

Comment:
@Rob--W is this an issue only in tests?

Reply:
Yes. The `process` global changed from an enumerable property to a non-enumerable one. This only affects iteration over globals, and the common normal use of accessing the `process` global is unaffected.

The `sandboxed-module` library iterates over the globals in order to discover globals that it needs to forward, here: https://github.com/felixge/node-sandboxed-module/blob/47fa6ccde05d7c48432d276f32c5f63ddb82bbc7/lib/sandboxed_module.js#L109-L111

Reply:
Thanks for the additional details, I see now.